src/router.js

```diff
@@ -80,15 +80,14 @@ export class Router {
   * @param viewPort The viewPort.
   * @param name The name of the viewPort. 'default' if unspecified.
   */
-  registerViewPort(viewPort:Object, name?:string) {
-    name = name || 'default';
+  registerViewPort(viewPort : Object, name? : string = 'default') : void {
```

Comment:
What could be a valid argument against default params is the code Babel generates.
```javascript 
function defaults(x = 'default') {
  console.log(x);
}
```
generates:
```javascript
function defaults() {
  var x = arguments.length <= 0 || arguments[0] === undefined ? 'default' : arguments[0];

  console.log(x);
}
```
Looking at this, perhaps we is better to use the 'old' way. What do you think?